Riot/Modules/People/Views/InviteRecentTableViewCell.m

```diff
@@ -79,22 +85,29 @@ - (void)onDeclinePressed:(id)sender
     }
 }
 
-- (void)onPreviewPressed:(id)sender
+- (void)onAcceptPressed:(id)sender
 {
     if (self.delegate)
     {
         MXRoom *room = roomCellData.roomSummary.room;
         
         if (room)
         {
-            [self.delegate cell:self didRecognizeAction:kInviteRecentTableViewCellPreviewButtonPressed userInfo:@{kInviteRecentTableViewCellRoomKey:room}];
+            [self.delegate cell:self didRecognizeAction:kInviteRecentTableViewCellAcceptButtonPressed userInfo:@{kInviteRecentTableViewCellRoomKey:room}];
```

Comment:
We still want to preview rooms. We want to directly accept only DM.

Reply:
It's handled like Android you need to tap background cell. Preview action in `[RecentsViewController tableView:didSelectRowAtIndexPath:]`